backend/urls.py

```diff
@@ -33,6 +33,13 @@
     path('api/', include(router.urls)),
     path('api/auth/', include('rest_framework.urls', namespace='rest_framework')),
 
+    # Redirect all of these to a login template page to display a message
+    path('accounts/login/', views.get_error_template, name='custom_login_view_accounts',
+         kwargs={'error_msg': "Account login is disabled as you need to launch this tool from an LTI tool provider"}),
+    path('admin/login/', views.get_error_template, name='custom_login_page_admin',
+         kwargs={'error_msg': "Admin login is disabled as you need to launch this tool from an LTI tool provider"}),
+    path('admin/logout/', views.get_error_template, name='custom_logout_view_admin',
+         kwargs={'error_msg': "Admin logout is disabled as you need to launch this tool from an LTI tool provider"}),
```

Comment:
Nit-pick: You could also put periods at the end of the sentences.

Reply:
All of these views could _probably_ have the same message. I put one in as the default. 

Or do you mean just to pass a partial like "Account Login" and "Admin Login" rather than the entire message? The get_error_template *might* be able to get the route then it'd doesn't need to be passed in separately and display like

`accounts/login is disabled as you need to launch this tool from an LTI tool provider.` I tried using default view like 
`defaults.page_not_found` but that would require us to provide a better 404 template page, otherwise there's no information on it. That seemed a little out of scope.

Reply:
Removed this in 7cc0bc93a4a708751d317e604719f4e92dc3785a